params/hopf_fibration.json

```diff
@@ -3,19 +3,18 @@
     "name": "Hopf Fibration",
```

Comment:
this one wasn't rendering for some reason, investigate it

Reply:
Ah right, the problem was that I was using a `box` that was made to be very thin. `quad` or `rand_box` are more appropriate for such cases.